libs/moloch-v3-data/src/utils/constants.ts

```diff
@@ -0,0 +1,45 @@
+export enum ProposalTypeIds {
```

Comment:
We probably need to review this move with @jordanlesich 

He was also having to do work around proposal types.